client/src/components/NavLinkDropdownMenu/NavLinkDropdownMenu.css

```diff
@@ -0,0 +1,14 @@
+.NavLinkDropdownMenu {
+  border: 2px solid gray;
```

Comment:
Can you use the CSS variable for this gray color?

Reply:
there's actually no gray border in the figma, but there is a box-shadow, so i'll remove the border and add the shadow